Source/ASDisplayNode+Subclasses.h

```diff
@@ -68,8 +68,22 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * @discussion This is the best time to add gesture recognizers to the view.
  */
+AS_CATEGORY_IMPLEMENTABLE
 - (void)didLoad ASDISPLAYNODE_REQUIRES_SUPER;
 
+/**
+ * An empty method that you can implement in a category to add global
+ * node initialization behavior. This method will be called by [ASDisplayNode init].
+ */
+AS_CATEGORY_IMPLEMENTABLE
+- (void)globalInit;
```

Comment:
This seems like a strange name for an instance method

Reply:
Hmm good point. `configure` or `setup` or even `didInit` or `didBaseInit`?

Reply:
Looks like the base class method is `_initializeInstance` which seems accurate and clear to me

Reply:
Renamed to `baseDidInit` and `baseWillDealloc`